book-example/src/for_developers/backends.md

```diff
@@ -171,6 +171,16 @@ Now imagine you don't want to count the number of words on a particular chapter
 usual `book.toml` configuration file by adding items to your `[output.foo]`
 table.
 
+```diff
+  [output.wordcount]
++ ignores = [
++     "For Developers",
++     "Preprocessors",
++     "Alternative Backends",
++ ]
++ deny-odds = true
+```
```

Comment:
this change seems unrelated, was it intended?

Reply:
Ah, sorry! You're right — that was an unrelated commit. I have reset to the earlier commit. Thanks!